.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+# Description
```

Comment:
You can commit a template with quoted material in it that won't be automatically shown when submitted. It might be useful to add some items here of a good example of a desription.

Additionally, I think these are good starting topics, but they are definitely tailored to our workflow. We could consider different questions in the future to improve the experience of a community member submitting a PR.